### PR TITLE
Indicator Component - React Version

### PR DESCRIPTION
**Pull request by FuturaExtraBold**
## Description
This adds a React version of the Indicator component.

## Notes
I added a proposed `preposition` prop for localization and/or developer preference. This has been applied to the Rails component, as well.

## Screenshots
image
image

### CODE REVIEW

File: packages/sage-react/lib/Indicator/Indicator.jsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+export const Indicator = ({
+  currentItem,
+  label,
+  numItems,
+  preposition,
```
Comment (FuturaExtraBold) [inline packages/sage-react/lib/Indicator/Indicator.jsx:9]: This is proposed for localization and/or developer preference. Thoughts on pros and cons are appreciated here.

**Review by philschanely — APPROVED** [review]
Looks good @FuturaExtraBold !

**Review by philschanely — APPROVED** [review]
Looking good!

**Review by anechol — APPROVED** [review]
🚀

File: docs/app/views/examples/components/indicator/_props.html.erb [hunk 2]
```diff
@@ -16,6 +16,12 @@
   <td><%= md('Integer') %></td>
   <td><%= md('`nil`') %></td>
 </tr>
+<tr>
+  <td><%= md('`preposition`') %></td>
+  <td><%= md('The preposition is between the two page numbers. This allows for words other than "of".') %></td>
+  <td><%= md('String') %></td>
+  <td><%= md('of') %></td>
+</tr>
```
Comment (joedakroub) [inline docs/app/views/examples/components/indicator/_props.html.erb:24]: This is a nice touch. 👍🏼

**Review by joedakroub — APPROVED** [review]
Very nicely done Ben! No concerns here. 👍🏼

File: docs/app/views/examples/components/indicator/_props.html.erb (hunk 2 above)
Comment (FuturaExtraBold) [reply docs/app/views/examples/components/indicator/_props.html.erb:24]: 😎

**Pull request merged**